src/sophi_hrt_pipe/hrt_pipe.py

```diff
@@ -611,32 +607,10 @@ def phihrt_pipe(input_json_file):
     if iss_off:
         print(" ")
         printc('-->>>>>>> Polarimetric Frames Registration (--> ISS OFF)',color=bcolors.OKGREEN)
-        limb_side, _, _ = limb_side_finder(data[:,:,0,cpos_arr[0],int(scan)], hdr_arr[int(scan)])
+        limb_side, _, _, sly, slx = limb_side_finder(data[:,:,0,cpos_arr[0],int(scan)], hdr_arr[int(scan)])
```

Comment:
but the limb_side_finder function now returns 'side' as the last value which you are not taking into account here?